File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int Height;
    int Space_amt;
    bool hasRun = false;
    do
    {

        Height = get_int("height: ");
        Space_amt = Height * 2;

    }
    while (Height < 1 || Height > 8);

    for (int i = 1; i < Height + 1; i++)
    {
        // row count is equal to i
        // your row count is HALF of what you need print.
        //  use \n at end of every row loop
        for (int y = Space_amt - 2; y > 0; y -= 2)
        {

            printf(" ");
        }

        hasRun = false;

        for (int j = 0; j < 2; j++)
        {

            for (int a = 0; a < i; a++)
            {

                printf("#");
            }

            if (hasRun == false)
            {
                printf("  ");
                hasRun = true;
            }

        }

        printf("\n");
        Space_amt -= 2;
    }
}

File: pset6/mario.py
from cs50 import get_int

Height = 0
space_amt = 0
hasRun = False



while(Height < 1 or Height > 8):
    Height = get_int("How many blocks tall? ")
    space_amt = Height*2

for i in range(1,Height + 1):
    for y in range(space_amt - 2, 0, -2):
        print(" ", end = "")

    hasRun = False

    for j in range(2):
        for a in range(i):
            print("#", end = "")
        if(hasRun == False):
            print("  ", end = "")
            hasRun = True
    print()
    space_amt -= 2


